src/features/creators/newScriptProject.ts

```diff
@@ -0,0 +1,18 @@
+import { MarkdownString, window } from 'vscode';
+import { PythonProject, PythonProjectCreator, PythonProjectCreatorOptions } from '../../api';
+import { PythonProjectManager } from '../../internal.api';
+
+export class NewScriptProject implements PythonProjectCreator {
+    public readonly name = 'newScript';
+    public readonly displayName = 'Script';
+    public readonly description = 'Create a new Python script project';
+    public readonly tooltip = new MarkdownString('Create a new Python script with basic structure');
```

Comment:
Similarly here, should we be presenting this as the base "Python project"

Reply:
so do you think I should just say "project" for the display name then too? Once we get pep 723 then we will look to separate them since the requirements will be done differently?

Reply:
also do we care these are the same wording:

    public readonly description = 'Create a new Python project';
    public readonly tooltip = new MarkdownString('Create a new Python project');